lighthouse-extension/gulpfile.js

```diff
@@ -60,8 +70,10 @@ const computedArtifacts = LighthouseRunner.getComputedGathererList()
 const locales = fs.readdirSync('../lighthouse-core/lib/i18n/locales/')
     .map(f => require.resolve(`../lighthouse-core/lib/i18n/locales/${f}`));
 
-const isDevtools = file => file.endsWith(CONSUMERS.DEVTOOLS);
-const isExtension = file => file.endsWith(CONSUMERS.EXTENSION);
+const isDevtools = file =>
+  file.endsWith(CONSUMERS.DEVTOOLS.src) || file.endsWith(CONSUMERS.DEVTOOLS.dist);
```

Comment:
can the `dist` file really ever go through this function?

Reply:
probably not but wanted to be save but i'm removing it